registry/reference.go

```diff
@@ -56,33 +56,65 @@ type Reference struct {
 	Reference string
 }
 
-// ParseReference parses a string into a artifact reference.
-// If the reference contains both the tag and the digest, the tag will be
-// dropped.
-// Digest is recognized only if the corresponding algorithm is available.
-func ParseReference(raw string) (Reference, error) {
-	parts := strings.SplitN(raw, "/", 2)
+// ParseReference parses a string (artifact) into an `artifact reference`.
+//
+// Note: An "image" is an "artifact", however, an "artifact" is not necessarily
```

Comment:
It looks like this sentence is incomplete.

Reply:
🤣 Fixed.